pages/android-app.md

```diff
@@ -24,11 +24,11 @@ Attention: the compass is very sensitive to magnetic materials. Android tries th
 
 (3) GPS position display toggle
 
-This will show a dot at your current position. In addition, in case an accuracy estimate is available, this shows a circle around the dot which indicates the positioning accuracy. The chance of your real position being in the circle is approximately 70%.
+If the currently active map work has valid global geographic coordinate system settings, this control will be enabled and will show a dot at your current position. In addition, in case an accuracy estimate (HDOP) is available, this shows a circle around the dot which indicates the positioning accuracy. The chance of your real position being in the circle is approximately 70%.
```

Comment:
> global geographic coordinate system settings

is not a term which is used anywhere in Mapper or its documentation. I would simply use "georeferencing" (maybe "geographic georeferencing").

Reply:
Makes sense. Usage of this term on my side likely stems from staring at the code, which frequently uses the acronym CRS. Will add a fix in a while.